backend/server-connection/src/main/java/org/sonarsource/sonarlint/core/serverconnection/storage/XodusServerIssueStore.java

```diff
@@ -192,8 +192,10 @@ private static ServerIssue adapt(Entity storedIssue) {
     var creationDate = (Instant) requireNonNull(storedIssue.getProperty(CREATION_DATE_PROPERTY_NAME));
     var userSeverity = (IssueSeverity) storedIssue.getProperty(USER_SEVERITY_PROPERTY_NAME);
     var type = (RuleType) requireNonNull(storedIssue.getProperty(TYPE_PROPERTY_NAME));
+    var impacts = readImpacts(storedIssue.getBlob(IMPACTS_BLOB_NAME));
```

Comment:
Be aware that for issues we have an incremental update with the pull mechanism. It means no impacts will be returned for all issues that are already stored. For such changes, we might want at some point to force a re-download of all issues. We had a similar case with [SLCORE-441](https://sonarsource.atlassian.net/browse/SLCORE-441)